### ISSUE TITLE
Extramacular location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11556]
Preferred term label:
Extramacular location

Synonyms

    Outside macula

    Extramacular region

    Sparing the macula

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization situated outside the macula lutea of the retina. This is frequently used to describe retinal deposits, drusen, lesions, or hemorrhages that are distributed in the retina but spare the central macula. PMID:21160312

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Extramacular drusen / Familial drusen (OMIM:126600)

    Diabetic retinopathy (OMIM:603933)

Your nano-attribution (ORCID)
0000-0001-9042-0508
